lib/phoenix_component.ex

```diff
@@ -771,10 +778,22 @@ defmodule Phoenix.Component do
   defp register_component!(kind, env, name, check_if_defined?) do
     attrs = pop_attrs(env)
 
+    # TODO: grab @doc and inject attr :desc
+    # at this point there is only one value for :doc
+    # so Module.get_attribute/2 should consistently work
```

Comment:
Something that I would also like to do is to allow something like `ATTRIBUTES`, in case someone wants to have explicit control over where attributes are inserted, for example:

```
## Attributes

ATTRIBUTES
```

We need a better placeholder but you get the idea. :)